<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>다인승 도로 단속 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 고딕</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>관리자 로그인</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
